opensuse_buildservice/notqmail-rpmlintrc

```diff
@@ -0,0 +1,21 @@
+addFilter(".* devel-file-in-non-devel-package")
+addFilter(".* unversioned-explicit-self-provides")
+addFilter(".* non-standard-gid")
+addFilter(".* is not allowed anymore in FHS 2.2.")
+addFilter(".* invalid-license .*")
+addFilter(".* permissions-world-writable .* /var/qmail/queue/lock/trigger is packaged with world writable permissions")
+addFilter("W: suse-filelist-forbidden-fhs23 /var/qmail is not allowed in FHS 2.3")
+addFilter("W: permissions-file-setuid-bit")
+addFilter("W: permissions-directory-setuid-bit .* is packaged with setuid/setgid bits .*")
+addFilter("W: permissions-missing-verifyscript missing %verify_permissions -e /var/qmail/bin/.*")
```

Comment:
Why do we need this? Can't we just call the suggested macro?

Reply:
I need to check this. IIRC, i had some issue using the macro on some version of openSUSE.

Reply:
This has been done and tested. The noperms variable has been removed to enable permissions.